pkg/controller/expcertificates/internal/secretsmanager/secret.go

```diff
@@ -46,31 +50,31 @@ type secretsManager struct {
 	enableSecretOwnerReferences bool
 }
 
-// secretData is a structure wrapping private key, certificate and CA data
-type secretData struct {
-	sk, cert, ca []byte
+// SecretData is a structure wrapping private key, Certificateificate and CA data
+type SecretData struct {
+	PrivateKey, Certificate, CA []byte
 }
 
-func newSecretsManager(
+func New(
 	kubeClient kubernetes.Interface,
 	secretLister corelisters.SecretLister,
-	certificateControllerOptions controllerpkg.CertificateOptions,
-) *secretsManager {
-	return &secretsManager{
+	CertificateificateControllerOptions controllerpkg.CertificateOptions,
```

Comment:
`CertificateificateControllerOptions` (also shouldn't have leading caps).
Can we change this to also just be `enableOwnerRef bool` given nothing else is used?